hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisClient.java

```diff
@@ -167,29 +166,25 @@ public OMResponse sendCommand(OMRequest request) throws ServiceException {
     CompletableFuture<RaftClientReply> raftClientReply =
         sendRequestAsync(request);
 
-    CompletableFuture<OMResponse> omRatisResponse =
-        raftClientReply.whenComplete((reply, e) -> LOG.debug(
-            "received reply {} for request: cmdType={} traceID={} " +
-                "exception: {}", reply, request.getCmdType(),
-            request.getTraceID(), e))
-            .thenApply(reply -> {
-              try {
-                // we need to handle RaftRetryFailure Exception
-                RaftRetryFailureException raftRetryFailureException =
-                    reply.getRetryFailureException();
-                if (raftRetryFailureException != null) {
-                  throw new CompletionException(raftRetryFailureException);
-                }
-
-                OMResponse response = OMRatisHelper
-                    .getOMResponseFromRaftClientReply(reply);
-
-                return response;
-              } catch (InvalidProtocolBufferException e) {
-                throw new CompletionException(e);
-              }
-            });
-    return omRatisResponse;
+    return raftClientReply.whenComplete((reply, e) -> LOG.debug(
+        "received reply {} for request: cmdType={} traceID={} " +
```

Comment:
Lets replace this with parameterized logging.

Reply:
But it is parameterized:

LOG.debug(
        "received reply {} for request: cmdType={} traceID={} " +
            "exception: {}", reply, request.getCmdType(),
        request.getTraceID(), e)